docs/docs/index.md

```diff
@@ -396,20 +338,6 @@ http_server if {
 
 <RunSnippet files="#input.json" command="data.servers.http_server"/>
 
-Providing good names for variables can be hard. If you only refer to the
```

Comment:
we'd prefer the some in style here so have removed, mostly to save reading time.